server/views/components/header/index.njk

```diff
@@ -2,14 +2,13 @@
   <div class="header__upper grid__cell">
     <div class="header__inner container">
       <div class="header__burger">
-        <button id="header-burger-trigger"
+        <a href="#footer-nav-1" id="header-burger-trigger"
           class="header__burger-trigger js-header-burger-trigger"
```

Comment:
We'll need to change the header styles.
At the moment without javascript the navigation is showing and the button is hidden